openshift-deps.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
is this one required ?

Reply:
It is not, I meant to remove it before submitting. I've pushed a commit to remove it.